src/main/java/io/getunleash/impactmetrics/GaugeImpl.java

```diff
@@ -0,0 +1,73 @@
+package io.getunleash.impactmetrics;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+class GaugeImpl implements Gauge {
+    private final String name;
+    private final String help;
+    private final ConcurrentHashMap<String, Long> values = new ConcurrentHashMap<>();
+
+    GaugeImpl(String name, String help) {
```

Comment:
in Node we accept MetricOptions that has 3 fields. why do we have 2 params here instead of the object param and how can we pass labelNames in the constructor as Node allows?